swagger/swagger.yaml

```diff
@@ -1640,6 +1640,121 @@ paths:
       responses:
         200:
           description: phenotypes varying within taxon
+  /taxon/annotations:
+    get:
+      tags:
+        - Taxa
+      summary: Taxon phenotype annotations
+      description: Query for taxon phenotype annotations.
+      produces:
+        - application/json
+        - text/tab-separated-values
+      parameters:
```

Comment:
It seems that the signature of the route also accepts a publication (which is good) but the documentation omits that parameter.

Reply:
That's right. I missed that. My bad.